src/main/java/com/aspectsecurity/contrast/contrastjenkins/ThresholdCondition.java

```diff
@@ -105,6 +112,24 @@ public FormValidation doCheckThresholdVulnType(@QueryParameter String value) {
             return FormValidation.ok();
         }
 
+
+        /**
+         * Validation of the 'applicationName' form Field.
+         *
+         * @param value This parameter receives the value that the user has typed.
+         * @return Indicates the outcome of the validation. This is sent to the browser.
+         */
+        public FormValidation doCheckApplicationName(@QueryParameter String value) {
+
+            if (value.isEmpty()) {
+                return FormValidation.error("Please enter an application name.");
```

Comment:
Please enter an application name -> Please enter an application name found in Teamserver.